sdk/examples/client/node_api_indexer/02_get_alias_outputs.rs

```diff
@@ -36,12 +36,12 @@ async fn main() -> Result<()> {
         ])
         .await?;
 
-    println!("Address output IDs {output_ids_response:#?}");
+    println!("Address output IDs: {output_ids_response:#?}");
```

Comment:
```suggestion
    println!("Alias output IDs: {output_ids_response:#?}");
```
You've changed to this in basic and foundry...